addons/mail/i18n/mail.pot

```diff
@@ -2374,9 +2374,11 @@ msgstr ""
 
 #. module: mail
 #. odoo-javascript
+#. odoo-python
 #: code:addons/mail/static/src/js/activity.js:0
 #: code:addons/mail/static/src/models/composer_view.js:0
 #: code:addons/mail/static/src/models/mail_template.js:0
+#: code:addons/mail/wizard/mail_compose_message.py:0
```

Comment:
Note that we don't really care about those information, so we could just update the python code ?

Reply:
Hello @tde-banana-odoo :wave:,

I did a test and those lines seem to be required for the string to be translated in the wizard.

When I remove those lines, install the "fr_BE" language, use that language, follow the steps mentioned in the commit message, the title of the composer switch from `Rédiger un email` to `Compose Email`. I don't have that issue when I keep those lines.

Reply:
Ok, I thought it was unnecessary :shrug: